Source/WebKit/GPUProcess/webrtc/RemoteSampleBufferDisplayLayer.cpp

```diff
@@ -167,6 +168,15 @@ void RemoteSampleBufferDisplayLayer::setShouldMaintainAspectRatio(bool shouldMai
     protectedSampleBufferDisplayLayer()->setShouldMaintainAspectRatio(shouldMaintainAspectRatio);
 }
 
+std::optional<SharedPreferencesForWebProcess> RemoteSampleBufferDisplayLayer::sharedPreferencesForWebProcess() const
+{
+    assertIsCurrent(m_consumeThread);
```

Comment:
This assertion fails on bots.
Note `RemoteSampleBufferDisplayLayer` is created on main thread, but message handlers are invoked on `WorkQueue` of `RemoteSampleBufferDisplayLayerManager` (see `RemoteSampleBufferDisplayLayerManager::dispatchMessage`); so m_consumerThread is main and current thread is not.
We might need to make `RemoteSampleBufferDisplayLayer` gets the `preferences` from its manager (whose `m_sharedPreferencesForWebProcess` can be accessed from `WorkQueue`),  by making `RemoteSampleBufferDisplayLayer` have a `ThreadSafeWeakPtr< RemoteSampleBufferDisplayLayerManager >` member.